File: subin/정렬/lv2_가장_큰_수.py
from functools import cmp_to_key

def compare(a, b):
    return int(b + a) - int(a + b);

def solution(numbers):
    strList = list(map(str, numbers))
    numbers = sorted(strList, key=cmp_to_key(compare))

    if numbers[0] == '0':
        return '0';
    else:
        return''.join(str(i) for i in numbers)

File: subin/정렬/lv2_가장_큰_수.js
function solution(numbers) {
  let answer = '';
  let sortArr = numbers
    .map((element) => String(element))
    .sort((a, b) => b + a - (a + b));

  if (sortArr[0] === '0') {
    return '0';
  } else {
    sortArr.map((element) => {
      answer += String(element);
    });
  }
  return answer;
}
